python/message_types.py

```diff
@@ -0,0 +1,36 @@
+""" Message Type Definitions, organized by class
+"""
+
+
+class CONN:
+    """ Connetion Class of Message Types.
+
+        This type notation is still being discussed and may change.
+    """
+
+
+class UI:
+    STATE = "urn:sovrin:agent:message_type:sovrin.org/ui/state"
```

Comment:
These message type strings are not consistent with the Message Types Indy HIPE. Should this be updated now, or is it using this older style string for compatibility?

Reply:
Good catch. I think we ought to update this now. I can't think of any reason to keep it around for compatibility purposes. I'll bring this up to @trthhrtz.